site/package.json

```diff
@@ -120,6 +120,7 @@
     "@types/react-html-parser": "^2.0.2",
     "@types/react-slick": "^0.23.7",
     "@types/tar": "^6.1.2",
+    "@types/tmp": "^0.2.3",
```

Comment:
Needed to fix missing typings in `tmp-promise` (if I remember package name correctly)